aff4/data_store.h

```diff
@@ -686,8 +692,8 @@ class MemoryDataStore: public DataStore {
 
     AFF4Status LoadFromTurtle(AFF4Stream& output) override;
 
-    AFF4Status Clear() override;
-    AFF4Status Flush() override;
+    virtual AFF4Status Clear() override;
```

Comment:
https://google.github.io/styleguide/cppguide.html#Inheritance

Explicitly annotate overrides of virtual functions or virtual destructors with exactly one of an override or (less frequently) final specifier. Do not use virtual when declaring an override. Rationale: A function or destructor marked override or final that is not an override of a base class virtual function will not compile, and this helps catch common errors.

Reply:
Right, so this is another point on which I find the Google C++ Style Guide to be entirely wrongheaded. Every `virtual` member function should be marked `virtual`, as this reminds the programmer that he is in fact dealing with a virtual function---as opposed to having to hunt through the class hierarchy to determine that.

Reply:
I disagree.  The `override` or `final` specifier is enough to let me know that it's a virtual function.  Those specifiers are not valid otherwise.

Reply:
I checked; I had not realized that the standard enforced that `override` implies `virtual`.  I've removed every `virtual` which co-occurred with an `override`.